### PR TITLE
Subaru: ES_Distance cruise cancel safety

### PR DESCRIPTION
**Pull request by jnewb1**
Current cancel request method (using simulated buttons) uses the same message that contains Cruise_Throttle meaning we could have sent any throttle we wanted

depends: https://github.com/commaai/panda/pull/1576

### CODE REVIEW

File: selfdrive/car/subaru/carcontroller.py [hunk 1]
```diff
@@ -64,7 +65,7 @@ def update(self, CC, CS, now_nanos):
         can_sends.append(subarucan.create_preglobal_es_distance(self.packer, cruise_button, CS.es_distance_msg))
 
     else:
-      if pcm_cancel_cmd and (self.frame - self.last_cancel_frame) > 0.2:
+      if pcm_cancel_cmd and (self.frame - self.last_cancel_frame)*DT_CTRL > 0.2:
```
Comment (jnewb1) [inline selfdrive/car/subaru/carcontroller.py:68]: this check didn't do anything before, since frames are whole numbers

Comment (adeebshihadeh) [inline selfdrive/car/subaru/carcontroller.py:68]: this should be done in a separate PR, it changes behavior

**Review by adeebshihadeh — CHANGES_REQUESTED** [review]

File: selfdrive/car/subaru/carcontroller.py (hunk 1 above)
Comment (jnewb1) [reply selfdrive/car/subaru/carcontroller.py]: done
